eessi_bot_event_handler.py

```diff
@@ -23,10 +24,25 @@
 
 from pyghee.lib import PyGHee, create_app, get_event_info, read_event_from_json
 from pyghee.utils import log
-import os
 
 
 class EESSIBotSoftwareLayer(PyGHee):
+    def __init__(self):
+        super().__init__()
+        cfg = config.read_config()
```

Comment:
Maybe we can use `self.cfg` here, and avoid having to read the configuration again later, just use `self.cfg` instead?

Reply:
This would actually fix #134